### PR TITLE
first draft of crest api documentation

### PR DESCRIPTION
**Pull request by Hootyoo**

### CODE REVIEW

File: crest.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ The loader exposes its current configuration in the local window, by setting a *
 
 - **config**: Contains configurations for Crest's operation.
 
-  - **crestConfig**: Specific configurations for Crest's behavior.
+  - **crestConfig**: Specific configurations for Crest's behaviour.
```
Comment (zmeggyesi) [inline crest.md:129]: Wasn't the agreement to write in US English?  
Not saying to go through the entire repo, but keep it in mind going forward.

**Review by zmeggyesi — CHANGES_REQUESTED** [review]
There's a lot of duplicated content here, the entire `crestStub` documentation is doubled up now. Please combine the new list with the list above it, so that we're not explaining everything twice twice.

File: crest.md (hunk 1 above)
Comment (zmeggyesi) [reply crest.md:129]: I would like at least the documentation to be consistent with the Dashboard - if one is US English, the other should be too.

The release notes I'm fine with being in UK English.

Comment (Hootyoo) [reply crest.md]: I also changed the other instance in this file that has been British for forever. There were no further behavio**u**rs in this repo that I could find.

**Review by zmeggyesi — APPROVED** [review]

**Pull request merged**